uninstall.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Function to display script usage
+function display_usage() {
+    echo "Usage: $0 [-f] <file>"
+    echo "  -f: Flag to force deletion jess with config files"
+    exit 1
+}
+
+# Default values
+force=false
+
+# Process command-line options
+while getopts ":f" opt; do
+    case "$opt" in
```

Comment:
minor thing: in bash use ${var} and not $var . Without {} that are tons of opportunities to do a mistake :)